src/mongocrypt.h

```diff
@@ -1530,6 +1531,33 @@ bool mongocrypt_ctx_setopt_query_type(mongocrypt_ctx_t *ctx, const char *query_t
 MONGOCRYPT_EXPORT
 bool mongocrypt_ctx_setopt_algorithm_range(mongocrypt_ctx_t *ctx, mongocrypt_binary_t *opts);
 
+/**
+ * Set options for explicit encryption with the "textPreview" algorithm.
```

Comment:
```suggestion
 * Set options for explicit encryption with the "textPreview" algorithm.
 * 
 * NOTE: "textPreview" is experimental only and may be removed in a future non-major release.
```

Suggested warning follows the warning above the `MONGOCRYPT_QUERY_TYPE_*` constants.